Src/Notion.Client/Models/Page/NewPage.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.Collections.Generic;
+
+namespace Notion.Client
+{
+    public class NewPage
```

Comment:
@asevos Needs your attention.

Reply:
@KoditkarVedant sorry, I am on a trip with no computer since Saturday. Will be back next Sunday. Till then I will not be able to edit code in my PR. 

I think builder class is unnecessary overcomplication. Right now both `NewPage`constructors have doc comments and I believe an SDK user would be able to figure out how to use it. It is not ideal, but I think it would work ok

Reply:
@asevos No problem. Enjoy your trip. I am merging your PR. Agree let's keep it simple for now.